app/services/report_template_format.rb

```diff
@@ -14,7 +14,7 @@ def self.selectable
   end
 
   def self.find(id)
-    all.find { |f| f.id.to_s == id.to_s }
+    all.find { |f| f.id.to_s == id.to_s.downcase }
```

Comment:
Are all report template names lowercase? I'd expect that nothing modifies IDs and users probably don't type IDs so I'm wondering why it's needed. **if** you're going to do this, I'd move `id.to_s.downcase` outside of the loop to avoid doing the same operation many times.

Reply:
@ekohl This one is just for the format (html, csv, etc.)

Reply:
It was so that the URL param `format=HTML` would work as well as `format=html`

Reply:
We generate the URLs and can use the correct format. Is that really important?